packages/shopify-app-session-storage-postgresql/src/postgresql.ts

```diff
@@ -176,4 +178,31 @@ export class PostgreSQLSessionStorage implements SessionStorage {
     if (row.expires) row.expires *= 1000;
     return Session.fromPropertyArray(Object.entries(row));
   }
+
+  private async hasLowercaseSessionTable(): Promise<boolean> {
+    const query = `
+      SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = $1
+    `;
+
+    // Allow multiple apps to be on the same host with separate DB and querying the right
+    // DB for the session table exisitence
+    const rows = await this.query(query, [
+      this.options.sessionTableName.toLowerCase(),
+    ]);
+    return Array.isArray(rows) && rows.length === 1;
+  }
+
+  private async migrateSessionTableV1_0_1(): Promise<void> {
```

Comment:
Ideally I merge my [PR](https://github.com/Shopify/shopify-app-js/pull/95) (hint hint for a review) and then you'll have to add this in the migrations file to be run as part of the new migration system :D